### PR TITLE
chore: added adept-augmentations to the spacy universe

### PR DESCRIPTION
**Pull request by davidberenstein1957**
added adept-augmentations to the spacy universe

## Description
Tom Aarsen and I created a cool package to create NER data augmentations using a low resource NER scenario to help with bootstrapping pipelines.

### Types of change
Documentation

## Checklist
- [X] I confirm that I have the right to submit this contribution under the project's MIT license.
- [X] I ran the tests, and all new and existing tests passed.
- [X] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/meta/universe.json [hunk 1]
```diff
@@ -2810,6 +2810,54 @@
             "tags": ["coreference", "multi-lingual", "cross-lingual", "allennlp"],
             "spacy_version": 3
         },
+        {
+            "id": "adeptaugmentations",
+            "title": "Adept Augmentations",
+            "slogan": " A Python library aimed at dissecting and augmenting NER training data for a few-shot scenario.",
+            "description": "EntitySwapAugmenter takes either a datasets.Dataset or a spacy.tokens.DocBin. Additionally, it is optional to provide a set of labels. It initially creates a knowledge base of entities belonging to a certain label. When running augmenter.augment() for N runs, it then creates N new sentences with random swaps of the original entities with an entity of the same corresponding label from the knowledge base. For example, assuming that we have knowledge base for PERSONS, LOCATIONS and PRODUCTS. We can then create additional data for the sentence \"Momofuko Ando created instant noodles in Osaka.\" using augmenter.augment(N=2), resulting in \"David created instant noodles in Madrid.\" or \"Tom created Adept Augmentations in the Netherlands\".",
```
Comment (bdura) [inline website/meta/universe.json:2817]: I'd suggest added backticks around code fragments as well as breaking the description into two paragraphs to make the it more readable.

```suggestion
            "description": "EntitySwapAugmenter takes either a `datasets.Dataset` or a `spacy.tokens.DocBin`. Additionally, it is optional to provide a set of labels. It initially creates a knowledge base of entities belonging to a certain label. When running `augmenter.augment()` for N runs, it then creates N new sentences with random swaps of the original entities with an entity of the same corresponding label from the knowledge base.\n\nFor example, assuming that we have knowledge base for `PERSONS`, `LOCATIONS` and `PRODUCTS`. We can then create additional data for the sentence \"Momofuko Ando created instant noodles in Osaka.\" using `augmenter.augment(N=2)`, resulting in \"David created instant noodles in Madrid.\" or \"Tom created Adept Augmentations in the Netherlands\".",
```

File: website/meta/universe.json [hunk 2]
```diff
@@ -2810,6 +2810,54 @@
...
+            "pip": "adept-augmentations",
+            "thumb": "https://raw.githubusercontent.com/Pandora-Intelligence/crosslingual-coreference/master/img/logo.png",
+            "code_example": [
+                "import spacy",
+                "from spacy.tokens import DocBin",
+                "",
+                "from adept_augmentations import EntitySwapAugmenter",
+                "",
+                "TRAIN_DATA = [",
+                "    \"Apple is looking at buying U.K. startup for $1 billion\"",
```
Comment (bdura) [inline website/meta/universe.json:2828]:
```suggestion
                "    \"Apple is looking at buying U.K. startup for $1 billion\",",
```

Comment (bdura) [reply website/meta/universe.json:2828]: Otherwise the two texts are merged, and you don't get the expected output.

**Review by bdura — CHANGES_REQUESTED** [review]
Hello @davidberenstein1957, thanks for the submission - it looks great! I suggested a few changes.

Could you also add a line that creates the pipeline? For instance:

```python
import spacy
from spacy.tokens import DocBin

from adept_augmentations import EntitySwapAugmenter

nlp = spacy.load("en_core_web_sm")

TRAIN_DATA = [
    "Apple is looking at buying U.K. startup for $1 billion",
    "Microsoft acquires GitHub for $7.5 billion"
]

# ...
```

Otherwise the example does not work out of the box.

**Comment by davidberenstein1957** [comment]
Thanks @bdura and good catch! I always struggle a bit with these `.json` code thingies.

**Comment by bdura** [comment]
@davidberenstein1957 this PR is in great shape. We can merge it as soon as you add the pipeline definition for a complete example.

**Comment by davidberenstein1957** [comment]
@bdura What do you mean by adding a pipeline definition for a complete example?

**Comment by bdura** [comment]
> @bdura What do you mean by adding a pipeline definition for a complete example?

The `nlp` object is not defined... We'd prefer the example to run as-is, for [instance by adding](https://github.com/explosion/spaCy/pull/12609#pullrequestreview-1419978880):

```python
nlp = spacy.load("en_core_web_sm")
```

**Comment by davidberenstein1957** [comment]
Ah I see now, I should have asked ChatGPT to rewrite my code to JSON-format.

**Review by bdura — APPROVED** [review]
Perfect!

**Pull request merged**